test/Microsoft.TemplateEngine.Core.UnitTests/ConditionalTests.CStyleEvaluator.cs

```diff
@@ -1395,7 +1395,10 @@ public void VerifyIfElseEndifConditionUsesFalse()
             Verify(Encoding.UTF8, output, changed, value, expected);
         }
 
-        [Fact(DisplayName = nameof(VerifyIfElseEndifConditionUsesDouble))]
+        [Fact(
+            DisplayName = nameof(VerifyIfElseEndifConditionUsesDouble),
+            Skip = "Temporarily disabled. The test does not pass in the RUS locale."
```

Comment:
Please add a link to the issue tracking the disabled test.

Reply:
done